schutzbot/deploy.sh

```diff
@@ -52,9 +52,12 @@ ARCH=$(uname -m)
 
 if [[ $ID == "rhel" && $VERSION_ID == "8.3" && -n "${RHN_REGISTRATION_SCRIPT:-}" ]] && ! sudo subscription-manager status; then
     greenprint "Registering RHEL"
-    sudo subscription-manager remove --all
     sudo chmod +x "$RHN_REGISTRATION_SCRIPT"
-    sudo "$RHN_REGISTRATION_SCRIPT"
+    for _ in 1 2 3 4 5
```

Comment:
I'd prefer if you changed this to something like:
```suggestion
    for _ in {0..4}
```
It looks better IMO, otherwise looks good, we'll see if that helps.